src/test/ui/suggestions/try-unimplemented.stderr

```diff
@@ -0,0 +1,20 @@
+error[E0277]: the trait bound `u32: std::ops::Try` is not satisfied
+  --> $DIR/try-unimplemented.rs:17:5
+   |
+17 |     3u32?
+   |     ^^^^^ `?` cannot be applied to a value of type `u32`
+   |
+   = help: the trait `std::ops::Try` is not implemented for `u32`
+   = note: required by `std::ops::Try::into_result`
+
+error[E0277]: the trait bound `(): std::ops::Try` is not satisfied
+  --> $DIR/try-unimplemented.rs:21:5
+   |
+21 |     Ok(3u32)?;
+   |     ^^^^^^^^^ cannot use the `?` operator in a function that returns `()`
```

Comment:
is this...aspirational?

Reply:
i.e., I don't see code that would emit this "cannot use" error

Reply:
It was intended to test `Return type does not implement Try`. Looking at `try-operator-on-main`, it has a pretty similiar error message, so maybe it was decided to use that error message instead.